packages/snowpack/src/config.ts

```diff
@@ -370,7 +370,10 @@ function loadPlugins(
 
   function loadPluginFromConfig(name: string, options?: any): SnowpackPlugin {
     const pluginLoc = require.resolve(name, {paths: [process.cwd()]});
-    const plugin = require(pluginLoc)(config, options);
+    const pluginRef = require(pluginLoc);
+    const plugin = pluginRef.default
+      ? pluginRef.default(config, options)
+      : pluginRef(config, options);
```

Comment:
Minor QoL improvement: I found it really easy to accidentally export a package with a `.default` export. Added this quick check which should make shipping plugins friendlier (TODO: plugin validation still needed!)

Reply:
+100000